src/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -720,40 +728,53 @@ public static void ExportUnrelatedPfx()
+                using (ImportedCollection ic = Cert.Import(exported))
+                {
+                    X509Certificate2Collection importedCollection = ic.Collection;
 
-                // Verify that the two collections contain the same certificates,
-                // but the order isn't really a factor.
-                Assert.Equal(collection.Count, importedCollection.Count);
+                    // Verify that the two collections contain the same certificates,
+                    // but the order isn't really a factor.
+                    Assert.Equal(collection.Count, importedCollection.Count);
 
-                // Compare just the subject names first, because it's the easiest thing to read out of the failure message.
-                string[] subjects = new string[collection.Count];
-                string[] importedSubjects = new string[collection.Count];
+                    // Compare just the subject names first, because it's the easiest thing to read out of the failure message.
+                    string[] subjects = new string[collection.Count];
+                    string[] importedSubjects = new string[collection.Count];
 
-                for (int i = 0; i < collection.Count; i++)
-                {
-                    subjects[i] = collection[i].GetNameInfo(X509NameType.SimpleName, false);
-                    importedSubjects[i] = importedCollection[i].GetNameInfo(X509NameType.SimpleName, false);
-                }
+                    for (int i = 0; i < collection.Count; i++)
+                    {
+                        subjects[i] = collection[i].GetNameInfo(X509NameType.SimpleName, false);
+                        importedSubjects[i] = importedCollection[i].GetNameInfo(X509NameType.SimpleName, false);
+                    }
 
-                Assert.Equal(subjects, importedSubjects);
+                    Assert.Equal(subjects, importedSubjects);
 
-                // But, really, the collections should be equivalent
-                // (after being coerced to IEnumerable<X509Certificate2>)
-                Assert.Equal(collection.OfType<X509Certificate2>(), importedCollection.OfType<X509Certificate2>());
+                    // But, really, the collections should be equivalent
+                    // (after being coerced to IEnumerable<X509Certificate2>)
+                    Assert.Equal(collection.OfType<X509Certificate2>(), importedCollection.OfType<X509Certificate2>());
+                }
             }
         }
 
         [Fact]
         public static void MultipleImport()
         {
             var collection = new X509Certificate2Collection();
-
-            collection.Import(Path.Combine("TestData", "DummyTcpServer.pfx"), null, default(X509KeyStorageFlags));
-            collection.Import(TestData.PfxData, TestData.PfxDataPassword, default(X509KeyStorageFlags));
-
-            Assert.Equal(3, collection.Count);
+            try
+            {
+                lock (Cert.SyncObject)
```

Comment:
Again, pretty sure the synchronization isn't valueable.  And overly aggressive locking in the tests just sends a message to customers that they need to synchronize their own cert creation.  I don't think we should do it.